CMakeLists.txt

```diff
@@ -12,25 +12,8 @@ find_package(Boost REQUIRED system filesystem)
 find_package(ASSIMP QUIET)
 if (NOT ASSIMP_FOUND)
   find_package(PkgConfig REQUIRED)
-  pkg_check_modules(ASSIMP assimp)
-endif()
-if (ASSIMP_FOUND)
```

Comment:
On Tue, Jul 05, 2016 at 02:00:03PM -0700, Dave Coleman wrote:
> MoveIt is now working on Kinetic with only two missing dependencies that are not in the ``ros-planning`` org - manipulation_msgs and household_objects_database_msgs. I think many groups are not using kinetic yet because projects like MoveIt! are not released into it yet. That's our job ;-)

I agree that we should release kinetic quickly! (Though I would really like to have the release with a unified moveit git repository)
My point was that we should still care about (as stable as possible) features and cleanups because indigo will also stay for some more time.
I was more talking about packages for hardware/robot support. Afaik the turtlebots are not yet fully supported, stuff like the pr2 kinematics package we are missing for the docs...